extensions/roulette/database/action.py

```diff
@@ -0,0 +1,23 @@
+from ..config import config
+from typing import Set
+
+
+class Role:
```

Comment:
Question: Is it possible to use discord.py's inbuilt `Role` class instead? There doesn't seem to be a significant reason to redefine one ourselves. Then `get_timeout_roles` can just become a public method (I'd also avoid using `fetch` since Discord.py uses that to imply API calls)

Reply:
While it's not really possible to use the `Role` class, I believe that using the `Object` class and passing the id would be the solution. (It works so far)

`return {Object(id=role_id) for timeout_role in timeout_roles}`

Reply:
Scratch that, it broke as soon as I posted the comment. I'll try to figure out why and possibly fix it